apps/backend/src/dynamodb.ts

```diff
@@ -13,15 +13,6 @@ import { CompanySchema, UserCompaniesSchema } from './db/schemas/CompanyUsers';
 
 dotenv.config();
 
-if (process.env.AWS_ACCESS_KEY_ID == null) {
```

Comment:
this is needed for dynamo access

Reply:
e1f9579a42a756a9d442c7797c9d3b9ef29e7537